tasks/image_configuration/set-fw-env-config/Kconfig

```diff
@@ -0,0 +1,7 @@
+config DS_GENERATE_FW_ENV_CONFIG
```

Comment:
It looks like this requires `DS_COMPONENT_TS_UBOOT_CONFIGS` in order to function correctly. I would recommend having this `select` `DS_COMPONENT_TS_UBOOT_CONFIGS`.

I would also recommend updating the `help` of `DS_COMPONENT_TS_UBOOT_CONFIGS` to make it more obvious that that option isn't necessarily useful on its own and is up to the end user to either enable `DS_GENERATE_FW_ENV_CONFIG` or they need to copy the appropriate `fw_env.conf` file to `/etc` in the final filesystem.